### PR TITLE
website(homepage changes): Callouts

### PR DESCRIPTION
**Pull request by EnMod**
[:mag: Preview Link](https://consul-hc9jwyucb-hashicorp.vercel.app/)

---

This PR adds an instance of `<Callouts />` to the homepage to highlight service mesh features.

### `TODO`
- [x] Fill in placeholder links

### CODE REVIEW

File: website/pages/home/index.jsx [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+      <Callouts
+        layout="two-up"
+        product="neutral"
+        items={[
+          {
+            icon: require('./img/kubernetes/logo.svg?include'),
+            heading: 'Consul Service Mesh on Kubernetes',
+            content:
+              'Use Helm to deploy and CRDs to configure Consul on Kubernetes.',
+            link: { text: 'Get started', url: 'https://consul.io' },
```
Comment (mwickett) [inline website/pages/home/index.jsx:25]:
```suggestion
            link: { text: 'Get started', url: 'https://learn.hashicorp.com/collections/consul/gs-consul-service-mesh' },
```

File: website/pages/home/index.jsx [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
+          },
+          {
+            icon: require('./img/kubernetes/communication-arrows.svg?include'),
+            heading: 'Consul as a Service Mesh',
+            content:
+              'Simplify, observe, and secure service to service communication for microservice architectures.',
+            link: { text: 'Read more', url: 'https://consul.io' },
```
Comment (mwickett) [inline website/pages/home/index.jsx:32]:
```suggestion
            link: { text: 'Read more', url: 'https://learn.hashicorp.com/collections/consul/developer-mesh' },
```

**Review by mwickett — APPROVED** [review]
Left suggestions with the proper links. Looking good!

**Comment by EnMod** [comment]
With approval and filled-in placeholders, I'm merging this in. Thanks @mwickett !

**Pull request merged**